### ISSUE TITLE
Website is down

### ISSUE THREAD

**Issue by hallmic1** [zedapp/zed#606]
zedapp.org is down for whois identification.

**Comment by poisson-myfish** [comment]
I was going to report that now. It seems they don't want to fix it?
Also, please edit your description, the real website is at zedapp.org

**Comment by xeddmc** [comment]
Yeah, I think the project might be abandoned :disappointed:

**Comment by hallmic1** [comment]
im afraid you're right, its a shame, zed is what I used when all I had was a chromebook. super useful

**Comment by zefhemel** [comment]
Yes. Zed is kinda abandoned. I stopped using it myself as I moved into management. And today with Chrome Apps being dead and alternatives like VS Code around, I’m not convinced it’s worth trying to keep this up.

**Comment by hallmic1** [comment]
I found the remote editor functionality really useful with Zed, just download a file and boom, you can remote edit, its a lot more lightweight than vs code is, especially when you don't have access to ssh

**Comment by zefhemel** [comment]
Right. Yeah that was one of the early use cases for me too. However indeed VS code now has something similar and more powerful, but indeed it requires direct SSH access whereas zedrem could even work from a private network with no inbound network connections. What can I say. I’m sure it would be possible to build zedrem type functionality as a VS Code add-on as well. I don’t have a use case for it, but I’m pretty sure it can be done.
